src/cls/rbd/cls_rbd.cc

```diff
@@ -3676,6 +3867,127 @@ int mirror_image_remove(cls_method_context_t hctx, bufferlist *in,
   return 0;
 }
 
+/**
+ * Input:
+ * none
+ *
+ * Output:
+ * @param std::vector<std::string>: collection of global_image_ids
+ * @returns 0 on success, negative error code on failure
+ */
+int mirror_image_status_list(cls_method_context_t hctx, bufferlist *in,
```

Comment:
It was used initially for `mirror pool status`, but is not used now. I will use your suggestion to batch up multiple get calls into a single op, and this method is most likely to be removed from the final version.